docs/configuration/features/vector_features.md

```diff
@@ -11,28 +13,29 @@ Preprocessing parameters:
 
 - `vector_size` (default `null`): size of the vector. If not provided, it will be inferred from the data.
 - `missing_value_strategy` (default `fill_with_const`): what strategy to follow when there's a missing value. The value should be one of `fill_with_const` (replaces the missing value with a specific value specified with the `fill_value` parameter), `fill_with_mode` (replaces the missing values with the most frequent value in the column), `fill_with_mean` (replaces the missing values with the mean of the values in the column), `backfill` (replaces the missing values with the next valid value).
-- `fill_value` (default ""): the value to replace the missing values with in case the `missing_value_strategy` is `fill_value`.
+- `fill_value` (default `""`): the value to replace the missing values with in case the `missing_value_strategy` is `fill_value`.
 
 ## Vector Feature Encoders
 
-The vector feature supports two encoders: `dense` and `passthrough`.  Only the `dense` encoder has additional parameters, which is shown next.
+The vector feature supports two encoders: `dense` and `passthrough`.
+
+### Passthrough Encoder
+
+There are no additional parameters for the `passthrough` encoder.
 
 ### Dense Encoder
 
-For vector features, you can use a dense encoder (stack of fully connected layers).
-It takes the following parameters:
+For vector features, a dense encoder (stack of fully connected layers) can be used to encode the vector.  It takes the
+following parameters:
 
 - `layers` (default `null`): it is a list of dictionaries containing the parameters of all the fully connected layers. The length of the list determines the number of stacked fully connected layers and the content of each dictionary determines the parameters for a specific layer. The available parameters for each layer are: `fc_size`, `norm`, `activation` and `regularize`. If any of those values is missing from the dictionary, the default one specified as a parameter of the encoder will be used instead. If both `fc_layers` and `num_fc_layers` are `null`, a default list will be assigned to `fc_layers` with the value `[{fc_size: 512}, {fc_size: 256}]` (only applies if `reduce_output` is not `null`).
```

Comment:
Check - is it called `layers` or `fc_layers`?  I think it should be `fc_layers` everywhere but not 100% sure.
```suggestion
- `layers` (default `null`): a list of dictionaries containing the parameters of all the fully connected layers. The length of the list determines the number of stacked fully connected layers and the content of each dictionary determines the parameters for a specific layer. The available parameters for each layer are: `activation`,
`dropout`, `norm`, `norm_params`, `output_size`, `use_bias`, `bias_initializer` and `weights_initializer`. If any of those values is missing from the dictionary, the default one specified as a parameter of the encoder will be used instead. If both `fc_layers` and `num_fc_layers` are `null`, a default list will be assigned to `fc_layers` with the value `[{output_size: 512}, {output_size: 256}]` (only applies if `reduce_output` is not `null`).
```